gem/templates/rwanda/core/main.html

```diff
@@ -5,23 +5,14 @@
     {% with cewekerenHost='cewekeren.com' ninyampingaHost='ninyampinga.com' qaHost='ninyampinga-rwanda.qa-hub.ie.gehosting.org'  %}
       {% if cewekerenHost in request.get_host %}
         <div class="freebasics-notification">
-          <p class="freebasics-notification__text">
-            {% blocktrans %}
-              Hei cewek keren! Apa kamu pelanggan Indosat? Kalo iya, kamu bisa mengakses
-              Girl Effect secara gratis lewat" %} <a href="http://www.freebasics.com" target="_top" class="freebasics-notification__anchor">Internet.org</a>
-              hanya dengan meng-klik pilihan "Girl Effect" yang tersedia di dalam menu. Keren banget, kan?
-            {% endblocktrans %}
+          <p class="freebasics-notification__text">Hei cewek keren! Apa kamu pelanggan Indosat? Kalo iya, kamu bisa mengakses Girl Effect secara gratis lewat<a href="http://www.freebasics.com" target="_blank" class="freebasics-notification__anchor">Internet.org</a> hanya dengan meng-klik pilihan "Girl Effect" yang tersedia di dalam menu. Keren banget, kan?
```

Comment:
should we have cerek keren in this one since these templates are just for RW?

Reply:
Updated...